fvm/src/gas/price_list.rs

```diff
@@ -286,6 +284,8 @@ lazy_static! {
             // Charge 0.4gas/byte for copying/fill.
             memory_copy_per_byte_cost: Gas::from_milligas(400),
             memory_fill_per_byte_cost: Gas::from_milligas(400),
+
+            host_call_cost: Gas::new(14000),
```

Comment:
So, the "wasm" prices are _generally_ just the prices used for wasm instrumentation. But the host call cost is so tightly bound to the wasm runtime that it kind of makes sense to just put it in this structure (and means that the syscall binding logic doesn't need to know about the global price list.

Reply:
You mean the VM (wasmtime)? We should, but we haven't bothered re-measuring them for a while.

But I should expand on this a bit more: The _rest_ of the prices here are actually baked into the compiled wasm modules when we instrument the code with gas accounting. That means they get cached in the engine (along with the compiled wasm modules) between epochs (across multiple FVM instances).

The host-call cost doesn't quite belong here because we charge for that on the outside so it doesn't become a part of the compiled wasm modules. Although, TBH, we _should_ charge for it inside wasm as we generally avoid "after the fact" charges like this.